settings.gradle.kts

```diff
@@ -1 +1,18 @@
 rootProject.name = "gradle-gcs-build-cache"
+
+enableFeaturePreview("STABLE_CONFIGURATION_CACHE")
+
+pluginManagement {
+    repositories {
+        mavenCentral()
+        gradlePluginPortal()
+    }
+}
+
+@Suppress("UnstableApiUsage")
+dependencyResolutionManagement {
+    repositoriesMode = RepositoriesMode.FAIL_ON_PROJECT_REPOS
```

Comment:
Why is this necessary?

Reply:
So this is not mandatory but this is recommended